components/channel_header/components/header_icon_wrapper.js

```diff
@@ -92,7 +93,7 @@ export default function HeaderIconWrapper({
                     trigger={['hover']}
                     delayShow={Constants.OVERLAY_TIME_DELAY}
                     placement='bottom'
-                    overlay={tooltip}
+                    overlay={isRhsOpen ? <></> : tooltip}
```

Comment:
How come this can't just return `null` when the RHS is open?

Reply:
It threw an error that it's not the right type of element for the overlay. 🤷‍♀

Reply:
That's odd, the type of `overlay` in `overlay_trigger.tsx` is `any`.
What's the error that you get?

Reply:
```react-hot-loader.development.js?c2cb:2408 Uncaught TypeError: Cannot read property 'props' of null
    at eval (overlay_trigger.tsx?1b61:29)
    at updateContextConsumer (react-dom.development.js?7f13:19771)
    at beginWork$1 (react-dom.development.js?7f13:20148)
    at HTMLUnknownElement.callCallback (react-dom.development.js?7f13:363)
    at Object.invokeGuardedCallbackDev (react-dom.development.js?7f13:412)
``` 

And I get it for all the components I updated

Reply:
Ohhh, it's because it's trying to read the props from the component you pass in...gotcha.
I wonder if there's like a null component you can use that provides those...otherwise I'm okay with this.